package.json

```diff
@@ -13,18 +13,26 @@
     "url": "https://github.com/plotly/react-plotly.js-editor/issues"
   },
   "scripts": {
+    "combine-translation-strings": "babel-node utils/combineTranslationKeys.js",
```

Comment:
`npm run make:combined-locales` ?

Just so it fits in with all the other make stuff?